Integration/XverifyIntegration.php

```diff
@@ -181,7 +181,7 @@ public function doEnhancement(Lead $lead)
                                     $status  = $this->getResponseStatus($response, $fieldKey);
                                     if (!is_null($status)) {
                                         $lead->addUpdatedField($fieldToUpdate, $status, null);
-                                        $persist = true;
+                                        //$persist = true;
```

Comment:
Accidental commit?

Reply:
not really, but a merge removed the other occurence I was checking for necessity and I didn't notice. Good catch,...this occurrence is now required and I will update my code.

Reply:
i should remove this however as it is not needed.